use-cases/using-Intel-SGX-and-kata.md

```diff
@@ -0,0 +1,68 @@
+# Kata Containers with SGX
+
+- [Install Host kernel with SGX support](#install-host-kernel-with-sgx-support)
+- [Install Guest kernel with SGX support](#install-guest-kernel-with-sgx-support)
+- [Run Kata Containers with SGX enabled](#run-kata-containers-with-sgx-enabled)
+
+Intel® Software Guard Extensions (SGX) is a set of instructions that increases the security
+of applications code and data, giving them more protections from disclosure or modification.
+At the time of writing this document, SGX patches have not landed on the Linux kernel
+project and `cloud-hypervisor` 0.8.0 doesn't support SGX, so specific versions of `cloud-hypervisor`,
+guest and host kernel must be installed to enable SGX.
+
+## Install Host kernel with SGX support
+
+The following commands were tested on fedora 32, they might work on other distros too.
+
+```sh
+git clone --depth=1 https://github.com/intel/kvm-sgx
+pushd kvm-sgx
+cp /boot/config-$(uname -r) .config
+yes n | make oldconfig
+# In the following step, enable: INTEL_SGX and INTEL_SGX_VIRTUALIZATION
+make menuconfig
+make -j$(($(nproc)-1)) bzImage
+make -j$(($(nproc)-1)) modules
+sudo make modules_install
+sudo make install
+popd
+sudo reboot
+```
+
+_Note: Run: `mokutil --sb-state` to check whether secure boot is enabled, if so, you will need to sign the kernel._
+
+Once you have restarted your system with the new brand Linux Kernel with SGX support, run
+the following command to make sure it's enabled. If the output is empty, go to the BIOS
+setup and enable SGX manually.
+
+```sh
+grep -o sgx /proc/cpuinfo
+```
+
+## Install Guest kernel with SGX support
+
+Install the guest kernel in the kata-containers directory, this way it can be used to run
+Kata containers.
+
+```sh
+curl -LOk https://github.com/devimc/kvm-sgx/releases/download/v0.0.1/kata-virtiofs-sgx.tar.gz
+sudo tar -xf kata-virtiofs-sgx.tar.gz -C /usr/share/kata-containers/
+sudo sed 's|kernel =|kernel = "/usr/share/kata-containers/vmlinux-virtiofs-sgx.container"|g' \
+/usr/share/defaults/kata-containers/configuration.toml
```

Comment:
I know this is awkward, but maybe we should copy the file to `/etc/kata-containers/` and then modify that version?